planet/src/main/webapp/places.css

```diff
@@ -12,6 +12,12 @@ header {
     padding:0; 
 }
 
+h3 {
+    margin: 10px !important;
+    font-size: 30px !important;
+    padding:10px !important;
+}
```

Comment:
It's better to avoid using "important". This part of the code would oblige all the "h3" elements including those of your teammates to follow this styling. You may wanna add them to a class!